test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -45,6 +45,8 @@ import (
 	"sigs.k8s.io/cluster-api/util/conditions"
 )
 
+var hookResponsesConfigMapName = "test-extension-hookresponses"
```

Comment:
How does this work together with Tilt? (I'm wondering about that we are using a different namespace here then in tilt, except if I missed something)

Reply:
Ah... 

It's not supposed to work together right? You just had to make changes in the test extension so that the test extension works with a static hook responses configmap and then you had to adjust the e2e test so that it still works?